### PR TITLE
done styling learningCurriculum page

### PR DESCRIPTION
**Pull request by ghost**
Styled the page with buttons and colors.

### CODE REVIEW

File: js/screens/LearningCurriculum/styles.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    alignItems: 'center',
+  },
+  subject: {
+    fontFamily: typography.secFont,
+    fontSize: 18,
+    padding: 10,
+  },
+  activity: {
+    fontFamily: typography.mainFont,
+    // color: colors.white,
```
Comment (siddharthparmar7) [inline js/screens/LearningCurriculum/styles.js:26]: Please remove this comment.

**Review by siddharthparmar7 — APPROVED** [review]
Thank you

**Review by siddharthparmar7 — CHANGES_REQUESTED** [review]
Your branch has merge conflicts. Please solve it. Thank you

**Comment by ghost** [comment]
solved

**Review by siddharthparmar7 — APPROVED** [review]

**Pull request merged**